pkg/platform/packet/template.go

```diff
@@ -84,6 +84,14 @@ module "packet-{{.Config.ClusterName}}" {
   }
   {{- end }}
 
+  {{- if .Config.ControllerCLCSnippets }}
+  controller_clc_snippets = [
+      {{- range .Config.ControllerCLCSnippets }}
+      "{{ . }}",
```

Comment:
Will that work for snippets, which are multi-line? I was usually using `<<EOF` in such cases.